src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,23 +1,48 @@
-import React from 'react';
+import { Dispatch, SetStateAction } from 'react';
+import { Filter } from '../../App';
+import { useDispatch } from 'react-redux';
+import { filterSlice } from '../../features/filter';
+
+type Props = {
+  filterType: string;
```

Comment:
The `filterType` prop is typed as a string. Ensure that it only takes values defined in the `Filter` enum to prevent any unexpected behavior.